soc/arm64/nxp_imx/imx8m/soc.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2020 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef SOC_H
+#define SOC_H
+
+#include <sys/util.h>
+#include <toolchain.h>
+
+#ifndef _ASMLANGUAGE
+
+#include <device.h>
+
+#endif /* !_ASMLANGUAGE */
+
+/* TODO: Place Holder */
```

Comment:
none seem to be required, please, justify them. Maybe some other files are not including necessary headers?